src/ages/age-one/getUserUnclaimedTokens.ts

```diff
@@ -24,23 +24,38 @@ export const getUserUnclaimedTokensFromDistribution = async (
     blockNumber,
   );
 
-  return userBalancesToUnclaimedTokens(userBalances.balances, endDate, epoch);
+  return userBalancesToUnclaimedTokens(address, userBalances.balances, endDate, epoch);
 };
 export const userBalancesToUnclaimedTokens = (
+  userAddress: string,
   balances: UserBalance[],
   endDate: BigNumber,
   epoch: keyof typeof configuration.epochs,
-) =>
-  balances
+) => {
+  let prevEpochRewards = BigNumber.from(0);
+
+  // add the previous epoch rewards to sum rewards for root computation
+  if (epoch === "epoch2") {
```

Comment:
`epoch` could be replaced with an `epochIndex` and thus we could sum up rewards from previous epochs but requires `age.epochs` to be an array as well as `epochResults`. A matter of another PR, but it'll be necessary to ease future maintainance